webplugin/js/app/kommunicate.js

```diff
@@ -191,9 +191,16 @@ $applozic.extend(true, Kommunicate, {
         KommunicateUI.showChat();
         KommunicateUI.hideFaq();
     },
-    openConversation: function (groupId) {
+    openConversation: function (groupId, groupDetail) {
         kommunicateCommons.setWidgetStateOpen(true);
-        window.$applozic.fn.applozic('loadGroupTab', groupId);
+        if (groupDetail && groupDetail.clientGroupId) {
```

Comment:
i think groupId condition should come first and then clientGroupId should be in else part if groupId is not present